lib/ansible/modules/cloud/azure/azure_rm_loadbalancer.py

```diff
@@ -784,7 +790,7 @@ def get_load_balancer(self):
         """Get a load balancer"""
         self.log('Fetching loadbalancer {0}'.format(self.name))
         try:
-            return self.network_client.load_balancers.get(self.resource_group, self.name)
+            return self.network_client.load_balancers.get(self.resource_group, self.name).as_dict()
```

Comment:
i'm not sure as_dict() whether works on load  balancer, but  it doesn't not work on webapp objects

Reply:
what happens in webapp?